src/main/java/org/openmrs/standalone/ReusableDB.java

```diff
@@ -0,0 +1,57 @@
+package org.openmrs.standalone;
+
+import ch.vorburger.exec.ManagedProcess;
+import ch.vorburger.exec.ManagedProcessException;
+import ch.vorburger.mariadb4j.DB;
+import ch.vorburger.mariadb4j.DBConfiguration;
+import ch.vorburger.mariadb4j.DBConfigurationBuilder;
+
+import java.io.File;
+
```

Comment:
Can you document what this class is used for?

Reply:
okay